engine_mgo.go

```diff
@@ -64,12 +64,13 @@ func (eng *mongoEngine) EnsureIndexes(collection Collection) {
 
 }
 
-func (eng *mongoEngine) Drop(collection Collection) {
+func (eng *mongoEngine) Drop(collection Collection) (bool, error){
```

Comment:
Added TODO. Don't know what kind of solution is going to be here eventually. I assune the mongo part is still in progress.